CONTRIBUTING.md

```diff
@@ -61,6 +61,14 @@ yarn build-editor
 yarn preview
 ```
 
+## Visual Regression
+
+To test if your changes make a difference visually across a large number of specifications, you can use can use the `yarn screenshot` command. 
+This command will use `dist/gosling.js` in a headless browser (using puppeteer) and take a screenshot of the visualization that is included in the editor. 
```

Comment:
You could briefly mention that one needs to run `yarn build` to test the updated Gosling.js.

Reply:
Will add!